util/src/Metrics.cpp

```diff
@@ -104,4 +104,84 @@ Counter Aggregator::GetCounter(const string& component_name,
                    component.values_.counters_);
 }
 
+// Generate a JSON string of all aggregated components. To save space we don't
+// add any newline characters.
```

Comment:
I was documenting it so that we don't add spaces in the string that we send over the wire. For example, if we pretty printed there would be spaces and newlines which would take up space in our 64k budget.